### ISSUE TITLE
InvalidOperationException: 'foo' already has a dependency defined for 'bar'

### ISSUE THREAD

**Issue by chriseldredge** [themotleyfool/Klondike#99]
Klondike fails to mirror some newer packages from nuget.org that probably have incompatible metadata with the version of NuGet.Core that Klondike currently uses.

```
2015-07-24 10:35:11,738 [Threadpool worker] INFO  NuGet.Lucene.Web.Models.MirroringPackageRepository - Mirroring package EntityFramework.Commands 7.0.0.0-beta5 from http://www.nuget.org/api/v2/
2015-07-24 10:35:11,832 [Threadpool worker] ERROR NuGet.Lucene.Web.UnhandledExceptionLogger - One or more errors occurred
System.AggregateException: One or more errors occurred ---> System.InvalidOperationException: 'EntityFramework.Commands' already has a dependency defined for 'EntityFramework.Relational.Design'.
  at NuGet.Manifest.ValidateDependencySets (IPackageMetadata metadata) [0x00000] in <filename unknown>:0
  at NuGet.Manifest.Validate (NuGet.Manifest manifest) [0x00000] in <filename unknown>:0
  at NuGet.Manifest.ReadFrom (System.IO.Stream stream, IPropertyProvider propertyProvider, Boolean validateSchema) [0x00000] in <filename unknown>:0
  at NuGet.Manifest.ReadFrom (System.IO.Stream stream, Boolean validateSchema) [0x00000] in <filename unknown>:0
  at NuGet.Lucene.FastZipPackage.ProcessManifest (System.IO.Stream manifestStream) [0x00000] in <filename unknown>:0
  at NuGet.Lucene.FastZipPackage.ProcessManifest (System.IO.Packaging.Package package) [0x00000] in <filename unknown>:0
  at NuGet.Lucene.FastZipPackage.Open (System.String fileLocation, System.IO.Stream stream, System.Byte[] hash) [0x00000] in <filename unknown>:0
  at NuGet.Lucene.FastZipPackage.Open (System.String fileLocation, System.Byte[] hash) [0x00000] in <filename unknown>:0
  at NuGet.Lucene.LucenePackageRepository.LoadStagedPackage (NuGet.Lucene.IO.HashingWriteStream packageStream) [0x00000] in <filename unknown>:0
```

**Comment by chriseldredge** [comment]
Fixed in https://github.com/themotleyfool/NuGet.Lucene/commit/7f620cacbb9103caf010d3d3ef35e8f4b3a0b166

**Comment by chriseldredge** [comment]
Fixed in 217dcb4134ef483d544c452cbf0b0a8a9ebeed9b / build 101.

https://ci.appveyor.com/project/chriseldredge/klondike/build/101/artifacts
